src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -78,15 +79,16 @@ def create_llm_inputs(
         input_filename: str = "",
         starting_index: int = DEFAULT_STARTING_INDEX,
         length: int = DEFAULT_LENGTH,
+        output_tokens_mean: int = DEFAULT_OUTPUT_TOKENS_MEAN,
+        output_tokens_stddev: int = DEFAULT_OUTPUT_TOKENS_STDDEV,
         prompt_tokens_mean: int = DEFAULT_PROMPT_TOKENS_MEAN,
         prompt_tokens_stddev: int = DEFAULT_PROMPT_TOKENS_STDDEV,
-        expected_output_tokens: int = DEFAULT_REQUESTED_OUTPUT_TOKENS,
         random_seed: int = DEFAULT_RANDOM_SEED,
         num_of_output_prompts: int = DEFAULT_NUM_PROMPTS,
         add_model_name: bool = False,
         add_stream: bool = False,
         tokenizer: AutoTokenizer = DEFAULT_TOKENIZER,
-        extra_inputs: Dict = {},
+        extra_inputs: Dict = None,
```

Comment:
Any reason for this change?

Reply:
Yes. This was my mistake originally. Using defaults for mutable types in parameters can lead to unexpected behavior. See [here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html#:~:text=Passing%20mutable%20lists%20or%20dictionaries,that%20the%20function%20is%20called.).

There were no issues because I did not update extra_inputs in the function, so CodeQL did not call it out. If someone changed the code to modify extra_inputs later, it would probably be caught, but I wanted to get rid of this now to avoid issues.

Reply:
Ah yes, I totally forgot that mutable objects should not be defaults in Python. In that case, I suggest we just update the type hints
```suggestion
        extra_inputs: Dict | None = None,
```